test/groovy/MTABuildTest.groovy

```diff
@@ -128,109 +85,138 @@ public class MTABuildTest extends BasePipelineTest {
 
         helper.registerAllowedMethod('pwd', [], { newDirPath } )
 
-        def mtarFilePath = mtaBuildScript.call(script: [commonPipelineEnvironment: cpe], buildTarget: 'NEO')
+        def mtarFilePath = mtaBuildScript.call(buildTarget: 'NEO')
 
         assert jscr.shell[0] =~ /sed -ie "s\/\\\$\{timestamp\}\/`date \+%Y%m%d%H%M%S`\/g" ".*\/newDir\/mta.yaml"$/
-
-        assert jscr.shell[1].contains("PATH=./node_modules/.bin:/usr/bin")
-
-        assert jscr.shell[1].contains(' -jar /opt/mta/mta.jar --mtar ')
-
         assert mtarFilePath == "${currentDir}/${newDirName}/com.mycompany.northwind.mtar"
-
-        assert jlr.log.contains("[mtaBuild] MTA JAR \"/opt/mta/mta.jar\" retrieved from environment.")
     }
 
+
     @Test
-    void mtaHomeNotSetTest() {
+    void mtaJarLocationNotSetTest(){
 
         new File("${currentDir}/mta.yaml") << defaultMtaYaml()
 
-        def mtarFilePath = mtaBuildScript.call(script: [commonPipelineEnvironment: cpe], buildTarget: 'NEO')
-
-        assert jscr.shell[0] =~ /sed -ie "s\/\\\$\{timestamp\}\/`date \+%Y%m%d%H%M%S`\/g" ".*\/mta.yaml"$/
+        helper.registerAllowedMethod('setupCommonPipelineEnvironment', [Map], {})
 
-        assert jscr.shell[1].contains("PATH=./node_modules/.bin:/usr/bin")
+        mtaBuildScript.call(buildTarget: 'NEO')
 
-        assert jscr.shell[1].contains(' -jar mta.jar --mtar ')
-
-        assert mtarFilePath == "${currentDir}/com.mycompany.northwind.mtar"
-
-        assert jlr.log.contains( "[mtaBuild] Using MTA JAR from current working directory." )
+        assert jscr.shell[1].contains('-jar mta.jar --mtar')
+        assert jlr.log.contains('[mtaBuild] Using MTA JAR from current working directory.')
```

Comment:
Keeping such nit changes out of the game here would de-blur the pull request and make the review more easy.